extension/src/popup/components/signTransaction/Operations.tsx

```diff
@@ -328,18 +362,18 @@ export const Operations = ({
               {operationIndex}. {OPERATION_TYPES[type] || type}
             </OperationBoxHeaderEl>
             <OperationsListEl>
-              {destination ? (
-                <>
-                  <KeyValueWithPublicKey
-                    operationKey="Destination"
-                    operationValue={destination}
-                  />
-                  <DestinationWarning
-                    destination={destination}
-                    flaggedKeys={flaggedKeys}
-                    isMemoRequired={isMemoRequired}
-                  />
-                </>
+              {account ? (
```

Comment:
the rest of this file is mostly alphabetizing the implementation of each key to match the params/types